src/App.vue

```diff
@@ -140,5 +137,11 @@ export default class App extends Mixins(DateMixin) {
   private onRouteChanged (): void {
     this.setCurrentStep(this.$route.meta.step)
   }
+
+  @Watch('tombStoneModel.entityType')
+  private onEntityTypeChanged (val:string | null) : void{
+    this.setCertifyStatementResource(val ? CertifyStatementResource
+      .find(x => x.entityType === val) : null)
+  }
```

Comment:
This is interesting! I can see this watcher ( or something similar) being expanded in the future to cover more than just the certify statement resource but also dictate what type of steps an entity has or anything else entity specific that could be handle by a resource file. Very cool.